domains/transfer/Transfer.API/API/Claiming/Api/Repositories/IClaimRepository.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using API.Claiming.Api.Models;
+
+namespace API.Claiming.Api.Repositories;
+
+public interface IClaimRepository
+{
+    Task<List<ClaimSubject>> GetClaimSubjects();
+    Task<ClaimSubject?> GetClaimSubject(Guid subject);
```

Comment:
Not sure I have a much better name, but I think the name `ClaimSubject` is confusing. Or it does not express the intent.

Reply:
It was lack of a better name